frame/collective/src/lib.rs

```diff
@@ -955,778 +926,4 @@ impl<
 	fn successful_origin() -> O {
 		O::from(RawOrigin::Members(0u32, 0u32))
 	}
-}
-
-#[cfg(test)]
-mod tests {
```

Comment:
just to be sure, only trivial change were made in the test isn't it ?

Reply:
at some point I would prefer not to do the refactor in this PR, so it is easier to review

Reply:
I assumed even pallet file structure needs to be refactored, sorry for my bad assumption :+1: ,

Shall I move the test back to `lib.rs` ? May be as suggested, will take file restructure part of  another PR